redcycle/admin.py

```diff
@@ -1,8 +1,31 @@
 from django.contrib import admin
-from .models.Account import *
+from django.contrib.auth.admin import UserAdmin
+from .models.User import *
 from .models.CommunityPost import *
 from .models.Location import *
+from .models.User import *
+from .forms.CustomUserChangeForm import *
+from .forms.CustomUserCreationForm import *
 
-admin.site.register(Account)
+class CustomUserAdmin(UserAdmin):
+    add_form = CustomUserCreationForm
+    form = CustomUserChangeForm
+    model = User
+    list_display = ('email', 'is_staff', 'is_active')
+    list_filter = ('email', 'is_staff', 'is_active',)
+    fieldsets = (
+        (None, {'fields': ('email', 'password')}),
+        ('Permissions', {'fields': ('is_staff', 'is_active')}),
+    )
+    add_fieldsets = (
+        (None, {
+            'classes': ('wide',),
+            'fields': ('email', 'password1', 'password2', 'is_staff', 'is_active', 'firstName', 'lastName')}
+        ),
+    )
+    search_fields = ('email',)
+    ordering = ('email',)
+
+admin.site.register(User, CustomUserAdmin)
```

Comment:
Adding this `CustomUserAdmin` object so that the admin dashboard can use our custom `User` object